packages/preview/src/preview.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+type PreviewElement = React.ElementRef<'div'>;
+type RootProps = React.ComponentPropsWithoutRef<'div'>;
+
+export interface PreviewProps extends RootProps {
+  text: string;
+}
+
+export const Preview = React.forwardRef<PreviewElement, Readonly<PreviewProps>>(
+  ({ text = '', ...props }, forwardedRef) => (
+    <div
+      ref={forwardedRef}
+      style={{
+        display: 'none',
+        overflow: 'hidden',
+        lineHeight: '1px',
+        opacity: 0,
+        maxHeight: 0,
+        maxWidth: 0,
+      }}
+      {...props}
+    >
+      {text ? (
+        <>
+          {text}
+          {renderWhiteSpace(text)}
+        </>
+      ) : (
+        ''
+      )}
```

Comment:
I don't think we need the condition here.

Reply:
good point